typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -337,12 +415,29 @@ export function updateTokenBalanceMetrics(
         token_name: token.name,
```

Comment:
these labels seem to be nearly the same between `warpRouteTokenBalance` and `warpRouteCollateralValue`. any reason to not have them be exactly the same? can we also do all this once and put the labels in a variable that are reused?